packages/eslint-config-eventbrite-react/rules/react.js

```diff
@@ -92,11 +92,9 @@ module.exports = {
         // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-max-props-per-line.md
         'react/jsx-max-props-per-line': ['error', {maximum: 4}],
 
-        // Prevent arrow functions & refs in a JSX prop (but still allow bind for now)
+        // Prevent `.bind`, arrow functions & refs in a JSX prop
         // https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md
-        'react/jsx-no-bind': ['error', {
-            allowBind: true,
-        }],
+        'react/jsx-no-bind': 'error',
```

Comment:
oOoo this is a big'n

Reply:
No kiddin'. Our apps are filled with this! But the nice thing is that we can (temporarily) turn the rule off in the apps themselves so we should be ok. New apps though should follow this